client/app/_components/layout/container/stories/Container.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Heading } from '@/_components/ui'
+import { Container } from '../Container'
+import type { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'layout/Container',
+  component: Container,
+  parameters: {
+    layout: 'fullscreen',
+  },
+  tags: ['autodocs'],
+} satisfies Meta<typeof Container>
+
+export default meta
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    children: null,
```

Comment:
Sorry also here, could you pass like <div>Children</div>? We won't pass null so it is better to show more realistic children props